modules/cnf-rfhe-notifications-api-refererence.adoc

```diff
@@ -0,0 +1,321 @@
+// Module included in the following assemblies:
+//
+// * networking/using-rfhe.adoc
+
+:_content-type: REFERENCE
+[id="cnf-rfhe-notifications-api-refererence_{context}"]
+= Redfish hardware event notifications REST API reference
+
+You can use the Redfish hardware fast event notifications REST API to subscribe an application to Redfish hardware events.
+
+Subscribe distributed unit (DU) applications to Redfish hardware events using the resource address `/cluster/node/<node_name>/redfish/event`, where `<node_name>` is the cluster node where Redfish compliant hardware is located.
+
+The Redfish events notifications REST API is served at `http://localhost:9087/` by default.
+
+[NOTE]
+====
+Only applications on localhost in the pod where the Redfish hardware event occurs can access the hardware events API on port 9087.
+====
+
+To get the name of the `hw-event-proxy` pod where you can access the Redfish hardware events API, perform the following procedure:
+
+* Get the pods running in the namespace where you installed the Hardware Event Proxy Operator, for example, `openshift-hw-events`:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-hw-events
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                                         READY   STATUS    RESTARTS       AGE
+hw-event-proxy-99bf99794-jvrxb                               3/3     Running   3              13d
+hw-event-proxy-operator-controller-manager-96fb6f55b-67nqp   2/2     Running   6              13d
+----
```

Comment:
I suggest remove this paragraph as it is misleading. Applications use the API on the application pod (with a cloud-event-proxy sidecar), not on hw-event-proxy pod.